### DISCUSSION TITLE
Flux CD notification with MS Teams

### DISCUSSION THREAD

**Discussion by saravanaaaa** [fluxcd/flux2#3930] [Q&A]
Hi,

Planning to setup the CD notifications for the pipeline with Flux notification controller and forward the notification to the teams channel with webhook. And we have added the notification yaml and made kustomization yaml watch it. But the flux cannot identify the yaml file and prompts the error.

Below is the example yaml which is used.

metadata:

  name: ms-teams

spec:

  type: ms-teams

  secretRef:

    name: ms-teams

  url: 

apiVersion: notification.toolkit.fluxcd.io/v1beta1

kind: NotificationPolicy

metadata:

  name: cd-failure

spec:

  eventSeverity: error

  receivers:

    - name: ms-teams

**Comment by somtochiama** [comment]
The formatting of the yaml isn't great but I can see a `kind: NotificationPolicy` which Flux doesn't have.
Please take a look at this guide for setting up notifications. https://fluxcd.io/flux/guides/notifications/
